invenio/templates/configurations/secrets.yaml

```diff
@@ -1,54 +1,59 @@
 ---
-{{- if not (.Values.rabbitmq.existing_secret) }}
+{{- if not (.Values.rabbitmq.invenio.existing_secret) }}
 apiVersion: v1
 kind: Secret
 type: Opaque
 metadata:
-  name: {{ .Values.rabbitmq.secret_name }}
+  name: {{ .Values.rabbitmq.invenio.secret_name }}
   labels:
-    app: {{ .Values.rabbitmq.secret_name }}
+    app: {{ .Values.rabbitmq.invenio.secret_name }}
   annotations:
     helm.sh/resource-policy: keep
 data:
-  RABBITMQ_DEFAULT_PASS: {{ .Values.rabbitmq.default_password | b64enc }}
-  CELERY_BROKER_URL: {{ .Values.rabbitmq.celery_broker_uri | b64enc }}
+  RABBITMQ_DEFAULT_PASS: {{ .Values.rabbitmq.auth.password | b64enc }}
+  CELERY_BROKER_URL: {{ (printf "amqp://%s:%s@%s:%s" .Values.rabbitmq.auth.username .Values.rabbitmq.auth.password (include "rabbitmq.host_name" .) (.Values.rabbitmq.service.ports.amqp | toString)) | b64enc }}
 
 ---
 {{- end -}}
-{{- if not (.Values.postgresql.existing_secret) }}
+{{- if not (.Values.postgresql.invenio.existing_secret) }}
 apiVersion: v1
 kind: Secret
 type: Opaque
 metadata:
-  name: {{ .Values.postgresql.secret_name }}
+  name: {{ .Values.postgresql.global.postgresql.auth.existingSecret }}
```

Comment:
Can we document this variable somehow more prominent? since we use `existing_secret` as a boolean for every other service. It is not clear that this is actually a string (secret name).

Reply:
You're right, it's confusing. I added a hint to README.md to clarify the difference